content/en/community/hackathons/sessions/testing-unikraft/work/prepare.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Same here as in the above `prepare.sh` script, please check it using [shellcheck](https://www.shellcheck.net/).

Reply:
Solved.